great_ape_safe/ape_api/univ3.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from brownie import interface, Contract
+from helpers.addresses import registry
+
+
+class UniV3:
+    def __init__(self, safe):
+        self.safe = safe
+
+        # contracts
+        self.NonfungiblePositionManager = safe.contract(
+            registry.eth.uniswap.NonfungiblePositionManager
+        )
+        self.pool = interface.IUniswapV3Pool(
+            registry.eth.uniswap.v3pool_wbtc_badger, owner=self.safe.address
+        )
+
+        # constant helpers
+        self.Q128 = 2 ** 128
+
+    def collect_fees(self):
+        """
+        loop over `TCL/positionData/` directory to grab each tokenID
+        to allow us to claim the fees earned on each range over time
+        """
+        path = os.path.dirname(f"scripts/TCL/positionData/")
+        directory = os.fsencode(path)
+
+        token0 = Contract(self.pool.token0())
+        token1 = Contract(self.pool.token1())
+
+        wbtc_bal_init = token0.balanceOf(self.safe.address)
+        badger_bal_init = token1.balanceOf(self.safe.address)
```

Comment:
if renaming these to `token0_bal_init` and `token1_bal_init`, the function makes more sense to be used on any uni v3 nfp.

Reply:
good suggestion, added in last commit